flink-java/src/test/java/org/apache/flink/api/java/operator/JoinOperatorTest.java

```diff
@@ -1211,7 +1210,6 @@ public String toString() {
 		public String myString;
 		public Object nothing;
 		public List<String> countries;
-		public Writable interfaceTest;
```

Comment:
Coverage was decreased only on the 1 line. But I returned to the previous version.
I did not expect that the unused variable may affect the coverage.